cnf-certification-test/identifiers/exceptions.go

```diff
@@ -49,5 +48,5 @@ const (
 	NoExceptions = `No exceptions`
 
 	// affiliated certification exception process
-	AffiliatedCert = NoDocumentedProcess + `Partner can run CNF Certification test suite before passing other certifications (Container/Operator/HelmChart) but the affiliated certification test cases in CNF Certification test suite must be re-run once the other certifications have been granted.` //nolint:lll
+	AffiliatedCert = NoDocumentedProcess + ` The partner can run the Red Hat Best Practices Test Suite before passing other certifications (Container/Operator/HelmChart) but the affiliated certification test cases in the Red Hat Best Practices Test Suite must be re-run once the other certifications have been granted.` //nolint:lll
```

Comment:
Did you mean a space before 'The'? Why is it 'the partner' and not 'a partner'?

Reply:
Yes, the space is meant. I can put it in a clearer way. I will also update the sentence with your suggestion.

Reply:
I see you used THE tool :-) which is perfectly fine. Just 'a partner' is better. @sebrandon1 knows for sure.

Reply:
Yup looks good to me.